crates/node-file-trace/src/lib.rs

```diff
@@ -487,24 +487,27 @@ async fn run<B: Backend + 'static, F: Future<Output = ()>>(
     let (sender, mut receiver) = channel(1);
     let dir = current_dir().unwrap();
     let tt = create_tt();
-    let console_ui = Arc::new(ConsoleUi::new(LogOptions {
-        current_dir: dir.clone(),
-        show_all,
-        log_detail,
-        log_level: log_level.map_or_else(|| IssueSeverity::Error, |l| l.0),
-    }));
     let task = tt.spawn_root_task(move || {
         let dir = dir.clone();
         let args = args.clone();
-        let console_ui = console_ui.clone();
         let sender = sender.clone();
         Box::pin(async move {
+            let console_ui = ConsoleUiVc::new(TransientInstance::new(LogOptions {
```

Comment:
This was moved here from the surrounding block to address the issue.

Reply:
Nit: move this down to where it's needed.